Src/Component.js

```diff
@@ -1,387 +1,277 @@
-function factory (global, factoryOpts) {
-    const messages = {
-        noElement: 'No element was specified while creating a instance of a Class. Creating a detached DOM Element instead.',
-        extendDeprecate: '@reduct/component.extend() is deprecated since v1.0.7 - Use the native ES6 extend instead.'
-    };
+/*global reductOpts*/
+
+import { logger } from '@reduct/logger';
+
+const componentLogger = logger.getLogger('@reduct/component');
+const messages = {
```

Comment:
I think that we should move the text resources into an own module and import them here:

```js
import TextResources from './TextResources';
```